ui/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@bitwarden/splunk-ui",
+  "version": "1.0.0",
```

Comment:
There is no point in maintaining so many versions. And since this is SPA, we do not bundle this version anywhere, so i think this is ok to leave it at whatever, like `1.0.0`. Let me know if you disagree.